README.adoc

```diff
@@ -112,34 +112,41 @@ include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Artist.j
 include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Album.java[tags=!comment]
 ----
 
-=== Using Jackson
-
-In the case of using Jackson for JSON binding, a few restrictions are applicable to these objects,
-they:
+=== Using JSON-B
 
-* Cannot have a constructor unless it is empty.
-* Must contain the same number of instance members as there are keys in the target JSON.
+JSON-B is a new feature introduced with Java EE 8 and strengthens Java's support for JSON.
+This guide previously used Jackson for JSON binding.  JSON-B requires, by default, a
+public default no-arguments constructor for deserialization.
```

Comment:
I've seen "no-argument" instead. Grammatically, the singular "argument" makes more sense to me, but please go with whatever is technically correct. ;)